rust/src/ops.rs

```diff
@@ -221,7 +224,7 @@ mod tests {
         // echo -n 012345678900cafe00deadbeef | xxd -r -p | sha256sum
         let expected = decode("0339f76086684506a6d42a60da4b5a719febd4d96d8b8d85ae92849e3a849a5e");
         assert!(
-            expected == apply_inner(&inner, &child).unwrap(),
+            expected == apply_inner::<HostFunctionsManager>(&inner, &child).unwrap(),
```

Comment:
It would be good to leave some CHANGELOG entry explaining the API change (use `xyz::<HostFunctionsManager>` rather than `xyz`) that work for the general std use cases. I know `ibc-rs` and penumbra are using this as well in std mode.